### ISSUE TITLE
My Falcon Remote no longer seems to be working.

### ISSUE THREAD

**Issue by dhollida55** [whitesoup12/remote-falcon-tracker#116]
My show is Hollidaylightshow.  I ran into some issues when I was trying to use other people's viewer pages that I downloaded from the gallery.  At first it seemed to be working until it stopped working.  So I thought I'd start from scratch.  I downloaded the default viewer page.  I make no changes to it and ran it as is.  I tried using it in Jukebox mode.  I set the limit of the queue to 10.  At first I was to pick some songs.  I picked 8 songs and it did indicate that there was 8 songs selected.  But it only displayed 6 songs, one in the Next Selection and only five in the Current Queue.  Also I noticed that I wasn't getting any kind of confirmation when I selected a song.  Do I need to turn on that messaging?  So I went into Remote Falcon and purged the Queue.  Then when I went back to my show it did indicate that the songs had been deleted I wasn't able to.  
Also I noticed on the Viewer Settings screen I no longer have the option to Restart Remote Falcon Plugin.  I should say that when I first started using the plugin there was an option to Automatically do a Restart when any changes were made.  But now I don't have that same field to turn off Automatic restarting.

**Comment by dhollida55** [comment]
I think I may have figured it out.  When resetting everything, to start from scratch I reset my location by accepting the value in the "Current Location Detected:" field.  So I clicked on that location to see where it located me and it was about 25 miles from where I am located.  So I found my actual location in Google Maps, entered that and now it seems to be working.  However I don't always get my confirmation or error messages.  eg. When I select a song to go in the Jukebox Queue I don't get the successful message.  Or when I exceed my Queue limit (trying to pick a 7th song when the limit is 6 songs) I don't get the error message.  So is there a problem with the lag time or something?
